### DISCUSSION TITLE
Category가 아닌 라벨을 이용해서 resolved 표시

### DISCUSSION THREAD

**Discussion by janeljs** [Sc0up/scoup-backend#156] [Convention]
카테고리를 이용하여 해결된 discussion을 resolved로 바꾸니까 resolved 이후 카테고리별 분류(e.g. Ideas, troubleshooting docs)를 알기 힘들다는 단점이 있는 것 같습니다. 카테고리는 그대로 두고 해결, 미해결 여부는 Label을 통해 관리하는 게 어떨까요?

**Comment by sallyjellyy** [comment]
그게 더 보기 편하겠네요!

**Comment by Dae-Hwa** [comment]
제가 생각했던건 discussion에서 resolve 된거에 대한 트러블 슈팅 문서를 만드는거라 생각했는데 ㅋㅋㅋ 내일 얘기해봐요!

**Reply by janeljs** [reply]
트러블 슈팅이 아니더라도 컨벤션이나 아이디어나 다 resolved로 바뀌어버리니까 카테고리가 제 역할을 못하는 것 같아서요ㅎㅎ

**Reply by Dae-Hwa** [reply]
컨벤션이 resolve 될 일이 있나요?? 컨벤션 기록해두기용이 아니었나보네요

**Reply by janeljs** [reply]
확정된 컨벤션은 노션에 따로 문서화하고 있으니까 discussions는 확정되지 않은 컨벤션 논의 또는 새로운 컨벤션 제안을 위한 것이라고 생각했어요. 그리고 토론 후 컨벤션이 확정되면 resolve된거라 생각했는데 서로 다르게 이해하고 있었군요ㅎㅎ
